trans/templates/home.html

```diff
@@ -151,8 +166,43 @@ <h3 style="color:green">
           <form
             class="form-inline"
             method="post"
-            action="{% url 'freeze_user_contest' username=user.username contest_id=contest.id%}">
+            action="{% url 'merge_print_and_freeze_contest' contest_id=contest.id%}">
             {% csrf_token %}
+            Select any other <b>EXTRA</b> country translations you want for your contestants and click Submit.<br><br>
+
+            Additional copy #1:
+            <select name="extra_country1">
+              <option value="NONE">None</option>
+              {% for user in trans_countries %}
+                <option value="{{ user.country.code }}">{{ user.country.name }} ({{ user.language.code }}_{{ user.country.code2 }})</option>
+              {% endfor %}
+            </select>
+            with
+            <input

```

Comment:
Instead of a text input, what if we have a dropdown, which has options from 0 to `num_of_contestants`? I think this will be nicer and clearer for the users.